### PR TITLE
Enable zip exports for tasks.

### PR DESCRIPTION
**Pull request by Rotonen**
* Create an upgrade step for the zipexport interface setting
* Add the interface
* Add tests for the export actions being there

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@ Changelog
 
 4.13.1 (unreleased)
 -------------------
+- Enable zip exports for tasks
```
Comment (phgross) [inline docs/HISTORY.txt]: Newline above is missing.

File: opengever/task/tests/test_overview.py [hunk 2]
```diff
@@ -57,13 +57,19 @@ def test_issuer_is_prefixed_by_current_org_unit_on_a_multiclient_setup(self, bro
 
     @browsing
     def test_date_of_completion_is_displayed_correclty(self, browser):
-        task = create(Builder("task").having(task_type='comment',
-                                             date_of_completion=date(2015, 02, 02),
-                                             issuer=TEST_USER_ID))
+        task = create(
```
Comment (phgross) [inline opengever/task/tests/test_overview.py]: We ignore the pyflakes length limit in tests. It just makes everything more unreadable in most of the cases.

File: opengever/task/tests/test_overview.py [hunk 3]
```diff
@@ -89,6 +95,41 @@ def test_documents_are_listed(self, browser):
             browser.css('#documentsBox a.document_link').text)
 
     @browsing
+    def test_zip_export_actions_are_available(self, browser):
+        dossier = create(
```
Comment (phgross) [inline opengever/task/tests/test_overview.py]: please use the following formatting when using the builder, as we use everywhere else:

```python
        task = create(Builder("task")
                      .within(dossier)
                      .titled(u'Aufgabe')
                      .having(text='Text blabla',
                              task_type='comment',
                              deadline=datetime(2010, 1, 1),
                              issuer=TEST_USER_ID,
                              responsible=TEST_USER_ID))
```

File: opengever/task/tests/test_overview.py [hunk 4]
```diff
@@ -89,6 +95,41 @@ def test_documents_are_listed(self, browser):
...
+            .within(task)
+            )
+
+        browser.login()
+        browser.open(task)
+        self.assertIsNotNone(
+            browser.css('.actionicon-object_buttons-zipexport'))
+
+        browser.open(task, view='tabbedview_view-documents')
+        self.assertIn('Export as Zip',
```
Comment (phgross) [inline opengever/task/tests/test_overview.py]: The action was already available before the change, but hasn't worked, so a test wich really made an zip-export of documents on a task, would be good.

File: opengever/task/tests/test_overview.py [hunk 5]
```diff
@@ -57,13 +57,19 @@ def test_issuer_is_prefixed_by_current_org_unit_on_a_multiclient_setup(self, bro
...
     @browsing
     def test_date_of_completion_is_displayed_correclty(self, browser):
-        task = create(Builder("task").having(task_type='comment',
-                                             date_of_completion=date(2015, 02, 02),
-                                             issuer=TEST_USER_ID))
+        task = create(
+            Builder("task")
+            .having(
+                task_type='comment',
+                date_of_completion=date(2015, 02, 02),
```
Comment (deiferni) [inline opengever/task/tests/test_overview.py]: i wouldn't use octal numbers here, people might start to wonder why `08` doesn't work 😉

**Comment by Rotonen** [comment]
@deiferni @phgross ready for rereview.

**Pull request merged**